craft_application/models/spread.py

```diff
@@ -140,14 +144,22 @@ class SpreadBackend(SpreadBaseModel):
     restore_each: str | None = None
     debug_each: str | None = None
 
+    # For the openstack backend
+    endpoint: str | None = None
+    account: str | None = None
+    key: str | None = None
+    location: str | None = None
+    plan: str | None = None
+    halt_timeout: str | None = None
```

Comment:
IIUC `halt_timeout` applies to all backends. What about the rest of these?

The main reason I ask is because of https://github.com/lengau/spread-schema